### PR TITLE
AMD device tree video, pcc and jtag updates

### PR DESCRIPTION
**Pull request by ojayanth**
This PR includes AMD platforms  congo, morocco, marley device updates related to video, jtag and PPC nodes.

### CODE REVIEW

File: arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-marley.dts [hunk 1]
```diff
@@ -49,17 +49,12 @@
...
-			size = <0x02000000>;	/* 32M */
-			alignment = <0x01000000>;
+		video_engine_memory0: jpegbuffer {
+			size = <0x0 0x04000000>;	/* 64M */
+			alignment = <0x0 0x00010000>;
 			compatible = "shared-dma-pool";
 			reusable;
 		};
-
-		pcc_memory: pccbuffer {
```
Comment (mahkurap) [inline arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-marley.dts:59]: why is pcc_memory deleted?

File: arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-marley.dts [hunk 2]
```diff
@@ -459,3 +453,7 @@
 &lpc0_uart_routing {
 	status = "okay";
 };
+
+&jtag0 {
```
Comment (mahkurap) [inline arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-marley.dts:457]: How does Tag2 with Yaapd work without this driver change?

**Review by mahkurap — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts [hunk 3]
```diff
@@ -49,18 +49,14 @@
...
-			size = <0x02000000>;	/* 32M */
-			alignment = <0x01000000>;
+		video_engine_memory0: jpegbuffer {
+			size = <0x04000000>;	/* 64M */
+			alignment = <0x0 0x00010000>;
 			compatible = "shared-dma-pool";
 			reusable;
 		};
-
-		pcc_memory: pccbuffer {
```
Comment (mahkurap) [inline arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts:59]: why pcc_memory node deleted?

File: arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts [hunk 4]
```diff
@@ -416,10 +412,13 @@
...
         A2600-15;
-        memory-region = <&pcc_memory>;
         rec-mode = <0x1>;
         port-addr-hbits-select = <0x1>;
         port-addr-xbits = <0x3>;
 
         status = "okay";
 };
+
+&jtag0 {
```
Comment (mahkurap) [inline arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts:422]: how does tag 2 work with yaapd without enabling the device tree?

Comment (mahkurap) [reply arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts:422]: Should we have this as a separate commit?

**Review by mahkurap — CHANGES_REQUESTED** [review]

**Review by mahkurap — COMMENTED** [review]
Same comments as in Marley/Congo device trees

**Comment by ojayanth** [comment]
Already added same support with different PR's

**Pull request closed without merging**